src/CreateSignalFactory.js

```diff
@@ -153,7 +153,24 @@ module.exports = function (controller, model, services, compute, modules) {
                   module: defaultOptions.modulePath.reduce(function (modules, key) {
                     return modules[key]
                   }, modulesArg)
-                })
+                }
+
+                if (utils.isDeveloping()) {
+                  try {
+                    actionFunc(actionArg)
+                  } catch (e) {
+                    action.error = {
+                      name: e.name,
+                      message: e.message,
+                      stack: actionFunc.toString()
+                    }
+                    controller.emit('signalError', {action: action, signal: signal})
```

Comment:
Can actually only get one signalError. So what happens is:

1. Signal starts and runs first action successfully
2. Signal continues on next action which throws an error
3. Cerebral picks it up, attaches it to the action, triggers and error event and then rethrows the exception
4. Cerebral just stops, because the current execution stack has thrown an error. But it has gotten a chance to inform the debugger about it